frontend/packages/metal3-plugin/src/plugin.ts

```diff
@@ -112,6 +112,17 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [FLAGS.BAREMETAL, METAL3_FLAG],
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: ['/k8s/cluster/nodes/'],
+      loader: async () =>
+        (await import(
+          './components/baremetal-nodes/BareMetalNodesPage' /* webpackChunkName: "node" */
```

Comment:
The `webpackChunkName: "node"` is already used for the Console nodes page.

```suggestion
          './components/baremetal-nodes/BareMetalNodesPage' /* webpackChunkName: "metal3-node" */
```

Reply:
Done

Reply:
When I change the chunk name to `metal3-plugin` the performance tests fail, saying that node chunk does not exist:
```
TypeError: Cannot read property 'decodedBodySize' of null
    at /go/src/github.com/openshift/console/frontend/integration-tests/tests/performance.scenario.ts:105:36
    at step (/go/src/github.com/openshift/console/frontend/integration-tests/tests/performance.scenario.ts:33:23)
    at Object.next (/go/src/github.com/openshift/console/frontend/integration-tests/tests/performance.scenario.ts:14:53)
    at fulfilled (/go/src/github.com/openshift/console/frontend/integration-tests/tests/performance.scenario.ts:5:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: Run it("downloads new bundle for node") in control flow
```
This is probably because the `metal3-node` chunk code imports node components which normally are part of `node` chunk and due to that webpack makes the `node` chunk part of `metal3-node` one.
I am inclined to fix this by calling the BareMetalNodesPage chunk `node`.